README.adoc

```diff
@@ -309,24 +400,94 @@ The default host name for Docker Desktop is `localhost`.
+The second URL adds properties from `system-service` to the inventory {kube} Service. 
+Visiting `{inventory-api}/[kube-service]` in general adds to the inventory
+depending on whether `kube-service` is a valid {kube} service that can be accessed.
+endif::[]
+
+// Cloud-hosted guide instruction
+ifdef::cloud-hosted[]
+Run the following command to get the node IP for the `system` service:
+
+```
+kubectl describe pod system | grep Node
+```
+{: codeblock}
+
+The output will show the node IP that will be used to access the service later. 
+It will appear in a format similar to the following:
+
+```
+Node:         10.114.85.140/10.114.85.140
+Node-Selectors:  <none>
+```
+
+Here the IP is `10.114.85.140` for the system service.
+Store the IP in a variable.
+
+```
+SYSTEM_HOST={system-node-ip}
+```
+{: codeblock}
+
+Use another `kubectl` command to get the node IP for the `inventory` service:
+
+```
+kubectl describe pod inventory | grep Node
+```
+{: codeblock}
+
+Store this IP in a variable as well.
+
+```
+INVENTORY_HOST={inventory-node-ip}
+```
+{: codeblock}
+
+Verify that the variables containing the IPs have been set correctly:
+
+```
+echo $SYSTEM_HOST && echo $INVENTORY_HOST
+```
+{: codeblock}
+
+Then use the following `curl` commands to access your microservices:
+
+```
+curl http://$SYSTEM_HOST:31000/system/properties
+```
+{: codeblock}
+
+```
+curl http://$INVENTORY_HOST:32000/inventory/systems/system-service
+```
+{: codeblock}
+
+The first URL returns system properties and the name of the pod in an HTTP header called `X-Pod-Name`.
+To view the header, you may use the `-I` option in the `curl` when making a request to the 
+`http://$SYSTEM_HOST:31000/system/properties` URL.
+The second URL adds properties from `system-service` to the inventory {kube} Service. 
+Making a request to the `http://$INVENTORY_HOST:32000/inventory/systems/[kube-service]` URL in general 
+adds to the inventory depending on whether `kube-service` is a valid {kube} service that can be accessed.
+endif::[]
 
 // =================================================================================================
 // Scaling a deployment
 // ================================================================================================
 
 == Scaling a deployment
 
-To use load balancing, you need to scale your deployments. When you scale a deployment, you replicate its pods, creating more running instances of your applications. Scaling is one of the primary advantages of {kube} because replicating your application allows it to accommodate more traffic, and then descale your deployments to free up resources when the traffic decreases.
+To use load balancing, you need to scale your deployments. 
+When you scale a deployment, you replicate its pods, creating more running instances of your applications. 
+Scaling is one of the primary advantages of {kube} because replicating your application allows it to accommodate more traffic, 
```

Comment:
Scaling is one of the primary advantages of {kube} because replicating your application allows it to accommodate more traffic, and then descale your deployments to free up resources when the traffic decreases.

--->

Scaling is one of the primary advantages of {kube} because you can replicate your application to accommodate more traffic,  and then descale your deployments to free up resources when the traffic decreases.